modules/lumble/client/init.lua

```diff
@@ -977,6 +978,14 @@ function client:requestUserList()
 	self:sendTCP(msg)
 end
 
+function client:requestACL(channel_id)
+	local channel_id = channel_id or 0 --default to root channel
+	local query = packet.new(13, proto.ACL())
```

Comment:
```suggestion
	local query = packet.new("ACL")
```

packet.new will automatically convert the string into a proper ID and whatnot, easier for readability.